computation-local/pom.xml

```diff
@@ -19,6 +19,10 @@
     <name>Local computation</name>
     <dependencies>
         <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
This dependency is not well sorted:
- dependency scope (compile, runtime, test)
- external dependency before internal dependency 
- groupId.artifactId